docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,9 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Introduced new optional ```RETURNING`` clause for :ref:`Update <ref-update>` to
+  return values based on each row actually updated.
+
```

Comment:
Isn't it possible to add `RETURNING` to the non-reserved keywords?

Otherwise I think we should also elaborate on the impact for users (That they'll have to quote "returning" if they use it as column name or alias)

Reply:
Good point, i will add it to the non-reserved keywords.

Reply:
and then move the change entry to non-breaking changes.